diff --git a/node_modules/@react-native-community/blur/src/BlurView.android.js b/node_modules/@react-native-community/blur/src/BlurView.android.js
index a192476..7fd03b6 100644
--- a/node_modules/@react-native-community/blur/src/BlurView.android.js
+++ b/node_modules/@react-native-community/blur/src/BlurView.android.js
@@ -4,9 +4,9 @@ import {
   View,
   requireNativeComponent,
   DeviceEventEmitter,
-  ViewPropTypes,
   StyleSheet,
 } from 'react-native';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types'

 const OVERLAY_COLORS = {
   light: 'rgba(255, 255, 255, 0.2)',
diff --git a/node_modules/@react-native-community/blur/src/BlurView.ios.js b/node_modules/@react-native-community/blur/src/BlurView.ios.js
index f675bf9..9cf1304 100644
--- a/node_modules/@react-native-community/blur/src/BlurView.ios.js
+++ b/node_modules/@react-native-community/blur/src/BlurView.ios.js
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import {View, requireNativeComponent, ViewPropTypes, StyleSheet} from 'react-native';
+import {View, requireNativeComponent, StyleSheet} from 'react-native';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types'

 class BlurView extends Component {
   setNativeProps = nativeProps => {
